src/main/java/nextstep/ladder/Line.java

```diff
@@ -0,0 +1,30 @@
+package nextstep.ladder;
+
+import java.util.List;
+import java.util.Random;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Line {
+
+    public static final int MAX_PERSON_SIZE = 1;
+    private List<Boolean> points;
+    private Random random = new Random();
+
+    public Line (int countOfPerson) {
+        if (countOfPerson <= MAX_PERSON_SIZE) {
+             throw new IllegalArgumentException("라인은 1명 이상인 경우만 생성 됩니다");
+        }
+        this.points = Stream.iterate(random.nextBoolean(), this::hasLine)
+                .limit(countOfPerson-1)
+                .collect(Collectors.toList());
+    }
+
+    private boolean hasLine(Boolean b) {
+        return b == true ? false : random.nextBoolean();
```

Comment:
아래와 같이 간소화 할 수 있을것 같아보입니다.
b는 정확히 의미하는바가 무엇인지 축약하지 말고 작성해주시면 좋을것 같습니다.
```suggestion
        return !b && random.nextBoolean();
```